app/assets/javascripts/discourse/app/models/post-stream.js

```diff
@@ -145,8 +145,12 @@ export default RestModel.extend({
   **/
   @discourseComputed("posts.[]", "stream.[]")
   previousWindow() {
+    if (!this.posts) {
+      return [];
+    }
+
     // If we can't find the last post loaded, bail
-    const firstPost = _.first(this.posts);
```

Comment:
the only thing first does is working with null/undefined.

```
_.first(null) // undefined
```

So I guess codepaths replacing _.first should ensure the object is not null/undefined.

Reply:
I believe I accounted for this. All tests are passing.